### PR TITLE
Add header sync status output to console

### LINKED ISSUE #62: output something when syncing headers
currently it just shows panda ascii and looks like it's not going anywhere

### PR DESCRIPTION
**Pull request by mr-pandabear**
Closes #62 and #63

### CODE REVIEW

File: blacklist.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # Enter one address per line in format http://<ip>:3000
-http://129.146.9.111:3000
-http://65.108.201.140:3000
\ No newline at end of file
+http://129.146.9.111:3000
```
Comment (johanaugustsandels) [inline blacklist.txt:2]: should be in a separate pr/commit

File: src/core/constants.hpp [hunk 2]
```diff
@@ -8,7 +8,7 @@
 #define TIMEOUT_SUBMIT_MS 30000
 #define BLOCKS_PER_FETCH 200
 #define BLOCK_HEADERS_PER_FETCH 2000
-#define BUILD_VERSION "0.5.5-beta"
+#define BUILD_VERSION "0.5.6-beta"
```
Comment (johanaugustsandels) [inline src/core/constants.hpp:11]: versions in separate commit

**Review by johanaugustsandels — APPROVED** [review]
minor nitpicks

File: blacklist.txt (hunk 1 above)
Comment (mr-pandabear) [reply blacklist.txt:2]: Will get these cleaner in future thanks!
